test/resources/demo-users/user/managed-ns1/rpa.yaml

```diff
@@ -16,14 +16,14 @@ spec:
     mapping:
       components:
         - name: sample-component
-          repository: "quay.io/gbenhaim0/managed/sample-component"
+          repository: "https://quay.io/repository/redhat-user-workloads/konflux-int-gitlab-tenant/sample-component/sample-component"
```

Comment:
No need to change the format of the link to quay.
But anyway, since we are not deploying this namespace, lets use a placeholder like in managed-ns2:
repository: <repository url>